src/labels/CustomLabels.labels

```diff
@@ -2015,6 +2015,70 @@
         <shortDescription>Recurring Donation Entry Form message when record is inserted</shortDescription>
         <value>Recurring Donation &quot;{0}&quot; was created.</value>
     </labels>
+    <labels>
+        <fullName>RD2_EntryFormPeriodAdvanced</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Picklist value for the RD2 Entry Form custom period picklist</shortDescription>
+        <value>Advanced</value>
+    </labels>
+    <labels>
+        <fullName>RD2_EntryFormPeriodHelpText</fullName>
+        <categories>Recurring-Donations, Entry-Form</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Inline Help Text for the Monthly and Advanced Picklist</shortDescription>
+        <value>Select &quot;Advanced&quot; to enter other schedule types such as weekly, yearly, or &quot;every 3 months&quot;.</value>
```

Comment:
@sam-knox Would you mind reviewing the custom help text for this picklist? This is the help text for the field with a label of "Recurring Period" (the label below this).

Reply:
@force2b - Sure thing. Use this for the help text:

"Select Advanced to enter other schedule types such as weekly, yearly, every 6 months, etc."

Reply:
Thanks @sam-knox. Change has been made.